ramls/instance.json

```diff
@@ -439,6 +439,11 @@
       "description": "Records the fact that the record should not be displayed in a discovery system",
       "default": false
     },
+    "deleted": {
+      "type": "boolean",
+      "description": "Indicates whether the record was marked for deletion",
+      "default": false
+    },
```

Comment:
The `instance-storage` interface minor version must be increased due to this change.